lib/travis/build/script/go.rb

```diff
@@ -81,6 +81,10 @@ def install
                 end
               end
             end
+            sh.elif '-f Gopkg.toml' do 
+              fetch_dep
+              sh.cmd 'dep ensure', retry: true, timing: true, assert: true, echo: true
```

Comment:
Where is this `dep` command supposed to come from?

Reply:
`fetch_dep` I believe

Reply:
Is this utility universally used? If not, it seems rather strange to let the presence of `Gopkg.toml` drive its use.

Reply:
`dep` is likely to be merged into the go toolchain, it's a more mature stab at how to handle dependencies:

https://github.com/golang/dep/wiki/Roadmap

A lot of people are starting to use it versus godep (which is currently on Travis-CI). The `Gopkg.toml` is basically like dep's version of a `package.json` or `pom.xml` file. If it's there, it means you are using `dep`.